### ISSUE TITLE
GUI on cbica-cluster doesn't work

### ISSUE THREAD

**Issue by chiharusako** [CBICA/CaPTk#831]
**Describe the bug**
captk GUI will not open on the cluster

**To Reproduce**
>module load captk/1.7.6
>captk

**Expected behavior**
GUI opens

**CaPTk Version**
/cbica/software/lab/captk/centos7/1.7.6/bin/

**Desktop (please complete the following information):**
cubic-login4

**Error message**

> captk
Checking for compatible OpenGL - this will happen only once.
QOpenGLFunctions created with non-current context
Segmentation fault

**Comment by AlexanderGetka-cbica** [comment]
notes: 
I tried this using the -X command line flag for ssh and still got this.
Furthermore, even glxinfo on the cluster gives an error. 

> [getkaa@cubic-login2 ~]$ glxinfo
name of display: 170.166.98.52:39.0
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  151 (GLX)
  Minor opcode of failed request:  24 (X_GLXCreateNewContext)
  Value in failed request:  0x0
  Serial number of failed request:  41
  Current serial number in output stream:  42

**Comment by AlexanderGetka-cbica** [comment]
@MarkBergman-cbica Do we have Slicer or MITKWorkbench installed on the cluster? I would like to compare to how these applications work on the cluster over ssh. It would also be useful for determining if the problem is on CaPTk's end or if it's related to openGL on the cluster.

**Comment by dboun** [comment]
ITK-SNAP seems to be there but I can't run it

```bash
[bouniasd@cubic-login2 ~]$ itksnap
Error running "/gpfs/fs001/cbica/software/lab/captk/centos7/1.7.2/lib/snap-3.8.0-beta/ITK-SNAP": Permission denied
```

**Comment by dboun** [comment]
Tagged as wontfix because  tt looks like the issue is incompatibility between our machines and the cluster, on the opengl level, and there isn't really anything we can do. Other machines work fine

**Comment by dboun** [comment]
Pinned this to hopefully disincentivize people from opening it again and again.
